docs/assignment1.md

```diff
@@ -1,62 +1,75 @@
 
-Don’t forget to use links to external URLs (e.g., the direct link to the Fantasy soccer league you are getting inspiration from, the link to the original video game you are getting inspiration from, etc.), if applicable. 
+After a plethora of discourse with the team, we have decided that our game, titled: **VUORK: Saving Thilo from the Chamber of Secrets**, will take place at the Vrije Universiteit Amsterdam. The user assumes the role of a newly enrolled student (user may choose the name) on his way to a 9AM lecture of Computer Programming. Albeit new, when the user arrives at the main campus of the VU, he notices something is wrong… This is where the game begins. 
 
-Maximum number of words for this section: 1000
+After a brief introduction into the game, the user will discover he is alone at the VU, and his lecturer, Thilo, is lost in the Chamber of Secrets. He will then have to gather the puzzle pieces, each with a clue indicating where the next piece of the puzzle is, to find the chamber of secrets; where Thilo is held captive. 
 
-## Features
-Author(s): `name of the team member(s) responsible for this section`
+To interact with the VUORK world, the user will use text commands in the CLI to give instructions. The instruction will be in the form “<action> <items…*>”. We will create a parser which will transform that instruction into something our system can understand: If the user types in a command, describing what he is doing/what he wants to do, he/she will get a response from the computer what she/he is now able to see, what has changed or if the move is not valid. Furthermore, instead of having a boring CLI as the realm, we want to add a few Graphical elements into the game. Examples include the players inventory, the current areas explored etc...
+
+With the game briefly explained, we must now consider how to develop this idea in a way that is easily extendable, maintainable and comprehensible. By using the Object-Oriented programming paradigm, we aim to create a system that is easily modifiable, expandable and intuitive for programmers to understand. To help achieve this goal, we will be making use of UML diagrams. 
 
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+   				   Figure 1: A use case diagram of VUORK
+
+![VuORK use case diagram](https://lh3.googleusercontent.com/ePg0EoRTAvMiKXEP_VNm-gGz7HN0pBql_kOfZ-RUjralgX5VEYfBDSGg7DdKa8HeYwpv4C79SSpxdnjOcfLk6GLn7E-IDENFk6257sC3hyDslfFupreykPBXSfHrvP1NY3T5XnxJ)
```

Comment:
Incorrect use of use case diagram. The use case diagram should represent on an abstract level the interaction between the user and the system. Here a mixture of class diagram is made